pkg/controller/podautoscaler/replica_calculator.go

```diff
@@ -208,7 +210,7 @@ func (c *ReplicaCalculator) calcPlainMetricReplicas(metrics metricsclient.PodMet
 
 	if len(missingPods) > 0 {
 		if usageRatio < 1.0 {
-			// on a scale-down, treat missing pods as using 100% of the resource request
+			// on a scale-down, treat missing pods as using exactly the target amount
 			for podName := range missingPods {
 				metrics[podName] = metricsclient.PodMetric{Value: targetUtilization}
```

Comment:
Why not max(target,100)?

Reply:
Here, these metrics are non-utlilization (non-percentage) metrics. These metrics here are the **usage** metrics like 10m CPU, 1G memory, etc.  Hence, making it equal to 100 would not be a good idea since this number (100) can be too big or too small depending on the metric.

Let say the target is 1G memory making it to = 100G memory (in case of missing metric) is not a good idea.

The variable `targetUtilization` here seems to be a bad copy paste from above function `GetResourceReplicas` and should be fixed, maybe to `targetUsage`

Reply:
Ok. Can we rename it in the next PR, please?